selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/workspaces/ProjectStateAfterRenameWorkspaceTest.java

```diff
@@ -94,9 +96,15 @@ public void checkProjectAfterRenameWs() throws Exception {
     dashboardWorkspace.clickOpenInIdeWsBtn();
     loader.waitOnClosed();
     seleniumWebDriver.switchFromDashboardIframeToIde();
-    projectExplorer.waitProjectExplorer();
-    projectExplorer.waitItem(PROJECT_NAME);
-    projectExplorer.waitItem(PROJECT_NAME + "/src/main/webapp/index.jsp");
+    try {
+      projectExplorer.waitProjectExplorer();
+      projectExplorer.waitItem(PROJECT_NAME);
+      projectExplorer.waitItem(PROJECT_NAME + "/src/main/webapp/index.jsp");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      Assert.fail("Known issue https://github.com/eclipse/che/issues/3574");
```

Comment:
It's better to localize exact single command which throws an exception because of issue.